src/main/java/core/basesyntax/services/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.interfaces.FileReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    private static final String EXCEPTION_MESSAGE = "Can't read file ";
+
+    @Override
+    public List<String> getLinesFromFile(String fileName) {
+        String path = fileName;
+        List<String> dataFromFile = new ArrayList<>();
+        File file = new File(path);
+        if (file.exists()) {
+            try {
+                dataFromFile = Files.readAllLines(Path.of(path));
```

Comment:
I think better to read line by line, because it can be null.
```suggestion
                 BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName));
            String line = bufferedReader.readLine();
            while (line != null) {
                System.out.println(line);
                dataFromFile.add(line);
                line = bufferedReader.readLine();
```

Reply:
I think, my code is right